psychopy/hardware/speaker.py

```diff
@@ -136,10 +185,10 @@ def createStream(self):
             # start with the rate from profile (this will usually work)
             self.profile['DefaultSampleRate'], 
             # if that fails, try some common sample rates
-            48000,
-            44100, 
-            22050, 
-            16000
+            48000.0,
+            44100.0, 
+            22050.0, 
+            16000.0
```

Comment:
Is this going to lead to the opposite problem - it'll fail when ptb is expecting an int? Should we just add both (seeing as we're silencing fails anyway)? Like:
```
48000,
48000.0,
44100,
44100.0,
etc.
```

Reply:
PTB returns floats for these values so I assumed it wants them back as that. I've been bitten by passing ints to extensions since they typically don't handle implicit conversion well.

Reply:
Anyways I restored the ints